Swift/ChartboostExample.xcodeproj/project.pbxproj

```diff
@@ -295,22 +334,22 @@
 					"@executable_path/Frameworks",
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = com.chartboost.ChartboostExampleSwift;
-				PRODUCT_NAME = Chartboost;
+				PRODUCT_NAME = "$(TARGET_NAME)";
```

Comment:
Why this change?

Reply:
yea I know, now the name is changed but it shows via the plist, and this isn't updated but is working, weird

Reply:
this was related adding the missing target back, it was already created the 'ChartboostExample' but somehow got unchecked to be used, so adding it back it changed this since having the two targets now

Reply:
i'm gonna revert it, and remove the last commit, should work